splinterd/packaging/splinterd.toml.example

```diff
@@ -13,44 +13,133 @@
 # limitations under the License.
 
 # Config file format version
-version = "1"
+#version = "1"
```

Comment:
This line (only this line) should be left uncommented since its metadata about the format of the file.